config/301-awseventbridgetarget.yaml

```diff
@@ -84,7 +84,7 @@ spec:
                       name:
                         type: string
               arn:
-                description: ARN of the EventBridge EventBus that will receive events. The expected format is documented at
+                description: ARN of the EventBridge that will receive events. The expected format is documented at
```

Comment:
Was that change intended? It doesn't look right. The resource is indeed called an "Event Bus".

To fix the linter warning, you can simply wrap the line, or remove "EventBridge" instead of "Event Bus" (since we know the source is about EventBridge already).